### ISSUE TITLE
Consistency with Julia implementation of SriMil

### ISSUE THREAD

**Issue by venkatachalapathy** [WheezePuppet/specstar#19]
Since Srikanth is working on a `julia` version of SriMil, getting our code working seamlessly with the Srikanth's version might be desirable.

**Comment by divilian** [comment]
Agreed, and Sri and I have been working towards this in principle, at least partially by both choosing Julia as the implementation language.

**Comment by divilian** [comment]
Commits 584b7bf and b9c12c0 are the first step towards implementing this.

**Comment by divilian** [comment]
@srikanthmudigonda: is the `fetch_neighbors()` function obsolete?

(Wait -- nm, I see that it's used in `form_possible_protos!()`.)

**Comment by divilian** [comment]
I'm calling this essentially done as of commit f4294eb.

**Comment by venkatachalapathy** [comment]
@WheezePuppet and @srikanthmudigonda, given that we had some confusion about the mismatch of proto-rules during our zoom meeting, I thought this issue should remain open.

**Comment by divilian** [comment]
I think this is essentially a duplicate of issue #31 (the only concrete example we know of) but we can keep this issue open as a placeholder too.
